applications/luci-app-csshnpd/htdocs/luci-static/resources/view/sshnpd/enroll.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+'require view';
+'require dom';
+'require fs';
+'require ui';
+'require uci';
+'require network';
+
+
+return view.extend({
+	handleCommand: function(exec, args) {
+		var buttons = document.querySelectorAll('.diag-action > .cbi-button');
```

Comment:
ES6 i.e. let

Reply:
Fixed in https://github.com/openwrt/luci/pull/7832/commits/9fd92bff9302ca6da3d3b4cba3650a75dc4298a5